tasks/main-clevis.yml

```diff
@@ -4,9 +4,9 @@
     name: "{{ __nbde_client_packages }}"
     state: present
 
-- name: Update clevis dracut config to enable networking if Tang binding is found
+- name: Create/Update nbde_client dracut config to enable networking if Tang binding is found
   lineinfile:
```

Comment:
We don't recommend the use of `lineinfile` - https://github.com/redhat-cop/automation-good-practices/tree/main/coding_style
I would suggest using the `file` module, creating a role `files/` subdirectory, and adding `nbde_client.conf` to the `files/` subdirectory.  If you think there will be other parameters to add to this config in the future, and the values of those configs will come from role variables, or be conditional, etc. then I suggest creating a template and using the `templates/` subdirectory.

Reply:
Thanks for the tip I'll go with templates since this may also assist with #22